src/Sylius/Bundle/FixturesBundle/Listener/ORMPurgerListener.php

```diff
@@ -76,20 +80,25 @@ protected function configureOptionsNode(ArrayNodeDefinition $optionsNode): void
 
         $optionsNodeBuilder
             ->enumNode('mode')
-                ->values(['delete', 'truncate'])
```

Comment:
I'm pretty sure we used to add another level of indention for node specification.